packages/visualizations/addon/templates/components/navi-visualizations/table.hbs

```diff
@@ -5,36 +5,48 @@
 </div>
 <div class="table-widget__horizontal-scroll-container">
   <div class= 'table-widget-container'>
-      {{#sortable-group class='table-header-row' direction='x' onChange='updateColumnOrder' as | group |}}
-        {{#each columns as |column|}}
+      {{#if isEditing}}
+        <div class='table-header-row'>
+          {{#each columns as |column|}}
+            <div class='table-header-cell {{column.type}}'>
```

Comment:
Have you looked at each of the column types to make sure the styles won't have any unintended consequences during edit mode.

Do we need the column type class during edit mode?

Reply:
It seems that the only difference is alignment